cirq/ops/controlled_gate.py

```diff
@@ -68,23 +118,27 @@ def _unitary_(self) -> Union[np.ndarray, NotImplementedType]:
         sub_matrix = protocols.unitary(self.sub_gate, None)
         if sub_matrix is None:
             return NotImplemented
-        return linalg.block_diag(np.eye(sub_matrix.shape[0]), sub_matrix)
+        return linalg.block_diag(
+                    np.eye(pow(2, self.num_qubits())-sub_matrix.shape[0]),
+                    sub_matrix)
 
     def __pow__(self, exponent: Any) -> 'ControlledGate':
         new_sub_gate = protocols.pow(self.sub_gate,
                                      exponent,
                                      NotImplemented)
         if new_sub_gate is NotImplemented:
             return NotImplemented
-        return ControlledGate(new_sub_gate)
+        return ControlledGate(new_sub_gate, self.control_qubits, # type: ignore
```

Comment:
Why is the `type: ignore` needed here?

Reply:
incompatible type "Tuple[None, ...]"; expected "Optional[Sequence[Qid]]"